configure-on-existing-repos.ps1

```diff
@@ -0,0 +1,35 @@
+# Loop through all local drives on the computer
+GET-WMIOBJECT -query "SELECT * from win32_logicaldisk where DriveType = '3'" | ForEach-Object {
+    $driveLetter=$_.DeviceID
+
+    Write-Output = "Scanning Drive: "$driveLetter
+    Get-ChildItem -Path $driveLetter\ -Filter .git -Recurse -ErrorAction SilentlyContinue -Force | Select-Object @{ n = 'Folder'; e = { Convert-Path $_.PSParentPath } } | ForEach-Object {
+        $repoDir=$_.Folder
+        # Go to the repository Folder
+        # Set-Location -Path $repoDir
+        
+        Write-Output "Installing hook in repo: $repoDir"
+        git secrets --install $repoDir
+        # git secrets --register-aws
```

Comment:
Why is this commented out?

Reply:
Because `configure-hooks-template.ps1` registers it globally, so we don't need to do it for each repo.

```
Write-Output "Registering AWS secret detection globally"
git secrets --register-aws --global
```